crates/ruff_linter/src/checkers/ast/analyze/statement.rs

```diff
@@ -591,18 +591,7 @@ pub(crate) fn statement(stmt: &Stmt, checker: &mut Checker) {
                 if checker.enabled(Rule::NonAsciiImportName) {
                     pylint::rules::non_ascii_module_import(checker, alias);
                 }
-                // TODO(charlie): Remove when stabilizing A004.
```

Comment:
Thanks @charliermarsh for adding the TODO comments. I would have missed this otherwise!